common/model-views.cpp

```diff
@@ -2954,6 +3054,14 @@ namespace rs2
             ImGui::Text("%s", msg.c_str());
             ImGui::PopStyleColor(2);
         }
+        else
+        {
+            if (_reflectivity_data_collected)
+            {
+                viewer.ref.reset();
+                _reflectivity_data_collected = false;
```

Comment:
Can't we make it so the Reflectivity class knows whether data has been collected (essentially, if it's not empty)?

Reply:
We can  I thought about it too but wasn't sure what I preffer , if you think its better I can change.

Reply:
Yes please

Reply:
Done